### PR TITLE
[security] update xterm.js to 3.9.2

### PR DESCRIPTION
**Pull request by marcdumais-work**

### CODE REVIEW

**Comment by marcdumais-work** [comment]
@benoitf  Thanks for keeping an eye out on our IP process compliance. The answer is no. A previous version of xterm was presumably covered earlier by this project-wide `node/npm` production dependencies [CQ](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=17349). In principle, when we change the version of such a production dependency, we should have the change approved by CQ. 

However ATM the Eclipse Foundation process to do so, for `production npm` dependencies, is very clunky: essentially we need a new CQ that covers `all production npm dependencies`, not only specific updated one(s), and it should cover the recursive tree of the dependencies of our dependencies, and their dependencies, and so-on. In other words, we would need to redo CQ #17349 above, every time we have a new `npm` production dependency or when any of the existing ones changes version, and then wait until it's approved before we are allowed to merge. 

I do not think that this approach will work for us or the Foundation, since adding or changing some production npm dependency is something that happens, if not daily, probably at least weekly, and will result in a lot of busy-work on both sides. 

The good news: @gorkem has been working from the inside to push for changes to better accommodate JS/TS projects with lots of prod dependencies, like ours. I do not know the all details yet, but I have a meeting late today with the Foundation to learn about it. 

In preparation, I am going to register a new `npm/node.js` CQ today, so that we are up-to-date w-r to our prod npm deps, and I hope we then have a sane way to keep the CQ updated when needed.

TL;DR: If this is PR is merged today, this new version of Xterm.js will be covered by the new CQ I will be registering.

**Comment by marcdumais-work** [comment]
merging this...
